code/drasil-example/ssp/lib/Drasil/SSP/IMods.hs

```diff
@@ -605,7 +615,8 @@ crtSlpIdQD = mkQuantDef fsMin crtSlpIdExpr
 crtSlpIdExpr :: Expr
 crtSlpIdExpr = apply minFunction [sy slopeDist,
```

Comment:
Out of curiosity, what needs to happen to make this right?

Reply:
I think what this "function" is supposed to do is to implement lots of the checks that are in the words that come with this instance model. Not sure what it is really supposed to return.